testsuite/oidc/rhsso/__init__.py

```diff
@@ -5,7 +5,7 @@
 from keycloak import KeycloakOpenID, KeycloakAdmin, KeycloakPostError
 
 from testsuite.oidc import OIDCProvider, Token
-from testsuite.objects import LifecycleObject
+from ...lifecycle import LifecycleObject
```

Comment:
this was done automatically, are we ok with this or should I change it to absolute path?

Reply:
I would always prefer full paths

Reply:
Would you also have an issue with first-level relative imports like `.auth_config`?

Reply:
I don't have a problem with "one-dot" imports with such a complex repo structure as ours :+1: